modules/nuxeo-apidoc-repo/src/main/resources/OSGI-INF/doctype-contrib.xml

```diff
@@ -6,7 +6,14 @@
       These contributions provide document types that handle persistence of introspected distributions.
     </documentation>
 
-    <doctype name="NXDistribution" extends="Folder">
+    <doctype name="NXExplorerFolder" extends="Document">
```

Comment:
I think lifecycles are not inherited so it would be ok to extend `Folder` here, no?

Reply:
Imagine a project where Folder document type would be removed: I'm trying not to rely on Folder doctype at all (even if it's not the exact problem I'm experiencing with current projects indeed)

Reply:
Ok